CONTRIBUTING.md

```diff
@@ -31,6 +31,30 @@ To execute only unit tests run `pytest`.
 
 To execute only integration tests run `pytest -m integration`.
 
+### Connection parameters for integration tests
+
+Parameters have to be set in environment parameters and must use the following format:
+
+``SNOWFLAKE_CONNECTIONS_INTEGRATION_<key>=<value>``
+
+where ``<key>`` is the name of the key
+
+For example: SNOWFLAKE_CONNECTIONS_INTEGRATION_ACCOUNT="my-account"
+
+List of required parameter keys:
+- account
+- user
+- password
+- host
+- port
```

Comment:
Is port really required? I think we can get a connection without it currently

Reply:
It is needed for local dev